src/utils/index.js

```diff
@@ -26,4 +28,28 @@ module.exports = {
     if (!isString(str)) return str;
     return str.toLowerCase();
   },
+
+  /**
+   * Returns the lowercased string if the string is valid.
+   * @param {string} str String to lowercase
+   * @return {string} Lowercased string
+   */
+  getAndInsertNames: async (address, DBHelper) => {
+    if (isUndefined(address)) return;
+    let addresses = address;
+    if (!isArray(address)) addresses = [address];
+    const toCheckAddresses = [];
+    await eachOfSeries(addresses, async (item, index) => {
+      const ret = await DBHelper.findOneName({ address: item });
+      if (!ret) {
+        toCheckAddresses.push(item);
+      }
+    });
+
+    const addressNameObj = await resolveAddress(toCheckAddresses);
+    Object.keys(addressNameObj).forEach((key) => (addressNameObj[key] == '') && delete addressNameObj[key]);
```

Comment:
lodash filter might be more efficient?

Reply:
don't know lodash filter how to filter like {"0xbc4b8726f9619c871fad66030116964480205b9d": "satoshi"}

Reply:
you should be able to use the iteration func, `filter(addressNameObj, (value, key) => addressNameObj[key]);`